docs/guides/js-waku/configure-discovery.md

```diff
@@ -52,6 +52,35 @@ const node = await createLightNode({
 });
 ```
 
+Alternatively, you can dial a particular node like this: 
+
+```js
+// Define the list of static peers to bootstrap
+const peers = [
+	"/ip4/0.0.0.0/tcp/60002/ws/p2p/16Uiu2HAkzjwwgEAXfeGNMKFPSpc6vGBRqCdTLG5q3Gmk2v4pQw7H",
+	"/ip4/0.0.0.0/tcp/60003/ws/p2p/16Uiu2HAmFBA7LGtwY5WVVikdmXVo3cKLqkmvVtuDu63fe8safeQJ",
```

Comment:
it may be worth including some info where to get these addresses

Reply:
added